tests/conftest.py

```diff
@@ -2153,50 +2255,94 @@ def moa_source44():
     }
 
 
-def _dict_check(expected_d: dict, actual_d: dict) -> None:
+def _dict_check(expected_d: dict, actual_d: dict, is_cdm: bool = False) -> None:
     """Make dictionary assertion checks. Check that actual matches expected data.
 
     :param expected_d: Expected dictionary
     :param actual_d: Actual dictionary
+    :param is_cdm: Whether checks are for transformers (CDM) or query handler.
+        CDM have extra fields that are not exposed to the query handler
     """
     for k, v in expected_d.items():
         if isinstance(v, dict):
-            _dict_check(v, actual_d[k])
+            _dict_check(v, actual_d[k], is_cdm=is_cdm)
         elif isinstance(v, list):
             actual_l = [json.dumps(v, sort_keys=True) for v in actual_d[k]]
-            expected_l = [json.dumps(v, sort_keys=True) for v in expected_d[k]]
+            if is_cdm:
+                expected_l = [json.dumps(v, sort_keys=True) for v in expected_d[k]]
+            else:
+                expected_l = []
+                for v in expected_d[k]:
+                    if isinstance(v, dict):
+                        if v.get("name") in {
+                            "therapy_normalizer_data",
+                            "disease_normalizer_data"
+                        }:
+                            updated_ext = v.copy()
+                            normalizer_data_type = v["name"].split("_normalizer_data")[0]  # noqa: E501
+                            updated_ext["name"] = f"{normalizer_data_type}_normalizer_id"  # noqa: E501
+                            updated_ext["value"] = v["value"]["normalized_id"]
+                            expected_l.append(json.dumps(updated_ext, sort_keys=True))
+                            continue
+                        else:
+                            new_extensions = []
+                            extensions = v.get("extensions") or []
+                            for ext in extensions:
+                                if ext.get("name") in {
+                                    "therapy_normalizer_data",
+                                    "disease_normalizer_data"
+                                }:
+                                    normalizer_data_type = ext["name"].split("_normalizer_data")[0]  # noqa: E501
+                                    new_extensions.append(
+                                        {
+                                            "name": f"{normalizer_data_type}_normalizer_id",  # noqa: E501
+                                            "type": "Extension",
+                                            "value": ext["value"]["normalized_id"]
+                                        }
+                                    )
+                                else:
+                                    new_extensions.append(ext)
+                            if extensions:
+                                v["extensions"] = new_extensions
```

Comment:
I don't love this. This was done because CDM expose more fields (this was needed for varcat) than exposed in the search endpoint. Because I wanted to get this out fast, I did not expose this additional information in the endpoint. I think this would require a plugin such as [APOC](https://neo4j.com/docs/apoc/current/) to help us access fields in a dict in cypher queries but didn't know if it would be difficult to setup on the ec2 instance.